source/retryable-reads/retryable-reads.rst

```diff
@@ -416,9 +425,11 @@ and reflects the flow described above.
       } catch (NetworkException networkError) {
         updateTopologyDescriptionForNetworkError(server, networkError);
         previousError = networkError;
+        previousServer = server;
```

Comment:
I don't think this is necessary, if a server fails with a network error (a transient error), then it's type should be set to Unknown ([docs](https://github.com/mongodb/specifications/blob/master/source/retryable-reads/retryable-reads.rst#why-are-read-operations-only-retried-once-by-default)). IIUC In this case, it is not possible for the server to be selected in the next retry ([docs](https://github.com/mongodb/specifications/blob/master/source/server-selection/server-selection.rst#topology-type-unknown)).